manager/volume.go

```diff
@@ -607,9 +607,13 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 		return nil, fmt.Errorf("cannot expand volume before replica scheduling success")
 	}
 
+	// volume shrinking is not supported the volume is already bigger then the requested size
```

Comment:
Do not remove the check here since users can use UI to expand a volume, which won't be prevented by Kubernetes.

Reply:
We don't support  shrinking, so if the user requests a smaller size then the volume already has, we can return success.
Since the wanted capacity is already covered by the available capacity.

Reply:
This also has the added benefit of making us compatible with the kubernetes expectation for volume expansion.

Reply:
I don't think it's a good idea...
1. If users try to shrink a volume, Longhorn will tell them it succeeds but with the size unchanged, which will confuse or mislead users.
2. On the other hand, the reason for volume shrinking may be that users want to save/reserve some space for other applications/places. But actually Longhorn won't release any spaces regardless of returning success.
3. Since Kubernetes will prohibit size shrinking, to make Longhorn be consistent with Kubernetes, we should error out as well. Here is the error message.
```
# persistentvolumeclaims "longhorn-simple-pvc" was not valid:
# * spec.resources.requests.storage: Forbidden: field can not be less than previous value
```

Reply:
IMO, the size in Longhorn is always the exposed block device's actual size, rather than a request. A PVC with a small size request could use a large PV, but the PV size is always accurate, right?